Module.md

```diff
@@ -0,0 +1,8 @@
+
+## Compatibility
+
+|                               |      	      Version             |
+| :---------------------------: | :-----------------------------: |
+|      Ballerina Language       |        Swan Lake Alpha 2  	  |
+|      Google People API        |                V1               |
+
```

Comment:
Now we need only package.md for central.
We can delete module.md

Reply:
Deleted